pennylane_lightning/core/src/simulators/lightning_tensor/utils/tncuda_utils/tncuda_helpers.hpp

```diff
@@ -53,4 +55,53 @@ inline SharedTNCudaHandle make_shared_tncuda_handle() {
     PL_CUTENSORNET_IS_SUCCESS(cutensornetCreate(&h));
     return {h, TNCudaHandleDeleter()};
 }
+
+/**
+ * @brief Returns the workspace size.
```

Comment:
Is it the number of bits?

Reply:
I guess it's number of bits, but it's not explicitly mentioned in the `cutensornet` docs.